test/visual/run-visual-tests.js

```diff
@@ -1,5 +1,5 @@
 'use strict';
-require('loud-rejection');
+require('loud-rejection'); // eslint-disable-line import/no-unassigned-import
```

Comment:
Actually, just noticed. This does nothing. It should either be executed as a function or imported as `loud-rejection/register`.